### DISCUSSION TITLE
水个帖

### DISCUSSION THREAD

**Discussion by ijog36** [share121/inter-knot#1552] [其他]
![1688657521343](https://github.com/user-attachments/assets/098aa671-7dbd-4e04-87ff-f3f2624472be)
绳匠你好，
你已通过治安官支援兵选拔，请于8月26日上午9:00 **前携带身份证到图书馆报告厅领取枪支。2024年8月27日9:00统一前往治安大队报道。在此我谨代表全国人民欢迎您的到来并感谢您的伟大付出届时你的名宇将永远载入史册，那是你无上的荣光。 再次提醒同学，前往治安官作战后续会开具综测证明，收到回复：学号＋姓名＋收到。
Привет, одноклассник
Вы прошли отбор российских бойцов поддержки, пожалуйста, принесите удостоверение личности в лекторий библиотеки к 9:00 26 марта, чтобы

**Comment by lllliufeng** [comment]
我去，新艾莉都治安官大学
新大nb！
